backend/src/database/queries/getSubGoals.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  mapIdToOntologyEntity,
+  parseOntologyEntityToQuery,
+  parsePrefixesToQuery,
+} from '../../common/database';
+import { PREFIXES } from '../index';
+
+export default (nodeId: string): string => {
+  const node = mapIdToOntologyEntity(nodeId);
+  if (!node) return '';
+
+  const fullNodeName = parseOntologyEntityToQuery(node);
+  const prefixString = parsePrefixesToQuery(PREFIXES.SDG, PREFIXES.SCHEMA, PREFIXES.RDFS);
+
+  return `
+      ${prefixString}
+      SELECT ?instancesOf
```

Comment:
This will only return instancesOf. Change to "SELECT ?instancesOf ?label ?description" to get all the info from the results.